### PR TITLE
Add assistance listing table and agency drill-down

### PR DESCRIPTION
**Pull request by danielnaab**
Implements issue(s) #2 

Changes proposed in this pull request:
- Add assistance listings table sources from beta.sam.gov
- Use assistance listings table to power a sub-agency drop-down on search form

### CODE REVIEW

File: distiller/audit_search/forms.py [hunk 1]
```diff
@@ -1,110 +1,119 @@
...
+    ('96', 'Social Security Administration'),
+    ('62', 'Tennessee Valley Authority'),
+    ('98', 'U.S. Agency for International Development'),
+    ('82', 'United States Information Agency'),
+    ('91', 'United States Institute of Peace'),
+    ('99', 'Miscellaneous'),
+)
 
+AGENCIES_BY_PREFIX = defaultdict(list)
+for choice in AGENCY_CHOICES:
```
Comment (cantsin) [inline distiller/audit_search/forms.py]: may be clearer to write as `for (id, agency) in AGENCY_CHOICES: ...`

File: distiller/audit_search/forms.py [hunk 2]
```diff
@@ -1,110 +1,119 @@
...
-    every two-digit combination between 00 and 99 is actually valid (i.e., listed
-    in FAC).
-    '''
-    return agency_prefix in AGENCIES_BY_PREFIX
+class AgencySelectionForm(forms.Form):
+    agency = forms.ChoiceField(choices=AGENCY_CHOICES)
+    sub_agency = forms.ChoiceField(required=False)
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
```
Comment (cantsin) [inline distiller/audit_search/forms.py]: Do we need this?

File: distiller/settings/production.py [hunk 3]
```diff
@@ -1,10 +1,12 @@
...
+"""
+
 from .base import *
 
 SECRET_KEY = os.environ.get('SECRET_KEY')
 
-# DEBUG is currently only turned on in dev settings, but better safe than sorry.
-DEBUG = True  # Revisit whether runserver is incompatible with collectstatic
-
+DEBUG = False
```
Comment (cantsin) [inline distiller/settings/production.py]: good catch :sweat_smile:

File: distiller/settings/base.py [hunk 4]
```diff
@@ -105,7 +113,7 @@
 STATICFILES_STORAGE = 'whitenoise.storage.CompressedManifestStaticFilesStorage'
 
 DATABASES = {
-    'default': dj_database_url.config(default='postgres://postgres:postgres@db/distiller')
+    'default': dj_database_url.config(default='postgres://postgres:postgres@127.0.0.1/distiller')
```
Comment (cantsin) [inline distiller/settings/base.py]: I haven't tested, but does changing the db host still work in Docker-land?

File: distiller/audit_search/forms.py (hunk 2 above)
Comment (danielnaab) [reply distiller/audit_search/forms.py]: Yes - we pass data into the constructor that needs to be registered with the parent class.

File: distiller/settings/base.py (hunk 4 above)
Comment (danielnaab) [reply distiller/settings/base.py]: I wasn't able to get it to work immediately, so punted for the moment. I'll try and revisit - we probably need to tweak the network config in `docker-compose.yml`.

**Pull request merged**